CHANGELOG.D/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
:smile: gitignore that ignores only itself

Reply:
According to towncrier manual — no. It deletes (and `git rm`'s) all other files when one runs `make changelog`